services/dispatcher/dispatcher.cpp

```diff
@@ -468,7 +468,7 @@ namespace services::dispatcher {
         collection_full_name_t key(index.database_, index.collection_);
         auto it_collection = collection_address_book_.find(key);
         if (it_collection != collection_address_book_.end()) {
-            make_session(session_to_address_, session_key_t{session, index.name()}, session_t(address, index));
+            make_session(session_to_address_, session_key_t{session, {}}, session_t(address, index));
```

Comment:
maybe change make_session(session_to_address_, session_key_t{session, {}}, session_t(address, index)); -> make_session(session_to_address_, session_key_t{session}, session_t(address, index)); ?

Reply:
I'm trying to do this. Looks like we have collision with multiple definition of make_session from collections. 
I've removed string name from key, but left struct.